tests/integration/static_ip_address_test.py

```diff
@@ -113,6 +133,42 @@ def test_remove_static_ipv4(setup_eth1_ipv4):
     assertlib.assert_state(desired_state)
 
 
+def test_remove_static_ipv4(setup_eth1_ipv4):
```

Comment:
This implies that the user input is: `state=up`.
But that with no addresses is an illegal input in IPv4.

Reply:
I guess you meant ipv4 (not ipv6).
`enable=False` is enough to disable.
`enable=True , address=[]` is illegal for ipv4, but legal for ipv6.

These are just reflections of the kernel behaviour.